src/_ctf-private/ctf-sign-up-banner/SignupBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Button, Container, Flex, Text, useBreakpointValue, useTheme } from '@chakra-ui/react';
+import Link from 'next/link';
+
+import { useSticky } from './hooks/useSticky';
+
+import CfLogo from '@icons/cf-logo.svg';
+
+export const SignupBanner = () => {
+  const isMobile = useBreakpointValue({ base: true, md: false });
```

Comment:
We can't use these checks reliably; the template is serverside rendered. Would you mind removing this and instead using `display: none` with breakpoints?